backend/models/Locations.ts

```diff
@@ -5,27 +5,35 @@ type ratingType = {
   dislikes: string[];
 };
 
+type comentType = {
+  author: string;
+  text: string;
+  likes: string[];
+  dislikes: string[];
+};
 export interface ILocation extends Document {
   locationName: string;
   coordinates: [number, number];
   photoSrc: string;
   description: string;
   rating: ratingType;
-  comments?: string[];
-  filters: string[]
+  comments?: comentType;
+  filters: string[];
+  author: string[];
 }
 
 const schema = new mongoose.Schema({
   locationName: { type: String, required: true },
   coordinates: [],
-  photoSrc: { type: String, required: false },
+  photoSrc: { type: Array, required: false },

```

Comment:
as it's an array, rename to "images" or smth like that

Reply:
done